filebrowser/run.sh

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env bash
 
 BOLD='\033[0;1m'
+
 printf "$${BOLD}Installing filebrowser \n\n"
 
-curl -fsSL https://raw.githubusercontent.com/filebrowser/get/master/get.sh | bash
+# Check if filebrowser is installed
+if ! command -v filebrowser &> /dev/null; then
```

Comment:
(No action, just commentary.)

This is nice, but unfortunately it's rare that users will benefit since it's usually installed to `/usr/local/bin`. For containers it's rarely persisted.

It'd be cool if we could store this in `$CODER_SCRIPT_BIN_DIR` instead. Usually this is within `/tmp` but it's possible to change by starting the coder agent with `CODER_AGENT_SCRIPT_DATA_DIR=/path/to/persistent/dir`.

This is something to consider for all modules, should we track this in a new issue @matifali?

Reply:
> This is nice, but unfortunately it's rare that users will benefit since it's usually installed to /usr/local/bin. For containers it's rarely persisted.

For air-gapped scenarios where the image already has it installed this is beneficial. 

> It'd be cool if we could store this in $CODER_SCRIPT_BIN_DIR instead. Usually this is within /tmp but it's possible to change by starting the coder agent with CODER_AGENT_SCRIPT_DATA_DIR=/path/to/persistent/dir.

I am a bit confused about $CODER_SCRIPT_BIN_DIR and $CODER_AGENT_SCRIPT_DATA_DIR. Do we have them already and how are the used? Or are you proposing this new concept.

Reply:
This is super useful and we should start using it in our modules.